Makefile

```diff
@@ -71,6 +71,7 @@ dependencies=go curl
 mocks=main/handlers/blockchain/mock/adapter_mock.go storage/mock/interfaces.go
 bindata=main/handlers/assets/bindata.go test/assets/bindata.go
 golocalimport=github.com/ProxeusApp/proxeus-core
+uiextraflags=NODE_OPTIONS=--openssl-legacy-provider
```

Comment:
It seems to me that it needs to be a MacOS, Node 17 only fix.

Reply:
[circleci](https://github.com/ProxeusApp/proxeus-core/blob/main/.circleci/config.yml) still running it with 16 node.js

maybe we should update our circleci config to be able to use this flag

Reply:
Let's upgrade CircleCI to 20.

Reply:
As discussed today, we are probably not quite ready for Node 20.x. I've updated this in #397